content-src/components/ContextMenu/ContextMenu.jsx

```diff
@@ -62,6 +62,17 @@ export class ContextMenuItem extends React.PureComponent {
     this.props.option.onClick();
   }
 
+  // This selects the correct node based on the key pressed
+  focusSibling(target, key) {
+    const parent = target.parentNode;
+    const closestSiblingSelector = (key === "ArrowUp") ? "previousSibling" : "nextSibling";
+    if (parent[closestSiblingSelector].firstElementChild) {
```

Comment:
I think you also need to return early if `parent[closestSiblingSelector]` is falsey for the first/last child case

Reply:
`if (!parent[closestSiblingSelector]) {
      return;
    } else if (parent[closestSiblingSelector].firstElementChild) {
      parent[closestSiblingSelector].firstElementChild.focus();
    } else {
      parent[closestSiblingSelector][closestSiblingSelector].firstElementChild.focus();
    }`

Does that look about right?

Reply:
The behaviour seems to be the same

Reply:
yep that's good, I see an error being thrown otherwise